### ISSUE TITLE
Uncaught DOMException

### ISSUE THREAD

**Issue by jjsarton** [jech/galene#233]
While issuing "/sendfile user" withi the chat we got DOMExeptions as well with Firefox as chromium:

```
Uncaught DOMException: Node.removeChild: The node to be removed is not a child of this node
    delFileStatusButtons https://galene.org:8443/galene.js:2685
    gotFileTransferEvent https://galene.org:8443/galene.js:2737
    event https://galene.org:8443/protocol.js:1807
    cancel https://galene.org:8443/protocol.js:1846
    fileTransfer https://galene.org:8443/protocol.js:2223
    onmessage https://galene.org:8443/protocol.js:507
    connect https://galene.org:8443/protocol.js:379
    serverConnect https://galene.org:8443/galene.js:4115
    <anonymous> https://galene.org:8443/galene.js:4024
```

**Comment by jjsarton** [comment]
> Which version of Galene is that? (Please type « git describe ».)

I use the actual galene version from github, branch master and have also tested this on galene.org:8443, same problem.

**Comment by jech** [comment]
Thanks, I've managed to reproduce it.  Interestingly, it does not occur when invoked from the menu, but does occur when invoked from the chat command.

**Comment by jech** [comment]
Fixed in 86eeb3d8cfd51fb97db19e66105c82d83625ec03.  Thanks.
